variables.tf

```
@@ -4,8 +4,10 @@ variable "cloudwatch_logs_retention_days" {
   type        = "string"
 }
 
-variable "environment" {
-  description = "Environment tag, e.g prod."
+variable "doc_url" {
+  default     = "https://example.com"
+  description = "URL for documentation on how to rotate keys."
+  type        = "string"
```

tf12 is starting to warn on this. 
```suggestion
  type        = string
```

---

Fixed this for all the variables in the module.